src/checkqueue.h

```diff
@@ -81,7 +81,7 @@ class CCheckQueue
                     fAllOk &= fOk;
                     nTodo -= nNow;
                     if (nTodo == 0 && !fMaster)
-                        // We processed the last element; inform the master he can exit and return the result
+                        // We processed the last element; inform the master he or she can exit and return the result
```

Comment:
Didn't take notice of the happening there... if it will be fixed I'm fine. It's likely better to not comment on the rest ^^.